x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/use_resource_findings.ts

```diff
@@ -98,6 +101,7 @@ export const useResourceFindings = (options: UseResourceFindingsOptions) => {
         assertNonBucketsArray(aggregations.clusterId?.buckets);
         assertNonBucketsArray(aggregations.resourceSubType?.buckets);
         assertNonBucketsArray(aggregations.resourceName?.buckets);
+        assertNonBucketsArray(aggregations.cloudAccountName?.buckets);
```

Comment:
Do `buckets` return as an empty array when data is not found? cause if it returns `undefined` we might get errors here

Reply:
Buckets returns an empty array.